interface/patient_file/encounter/new_form.php

```diff
@@ -49,7 +49,6 @@ function toggleFrame1(fnum) {
 var oldddmenuitem = 0;
 var flag = 0;
 
-// open hidden layer
 function mopen(id)
 {
     // cancel close timer
```

Comment:
just pulled you code cant seem to reproduce the dropdown :cold_sweat:  taking a closer look

Reply:
Actually the real change was made in pr made at #1200 please take a look at that. I made this pr because I created the old one and merged to master branch so I was asked to put it in a different branch hence. 😅

Reply:
okay will pull that and test

Reply:
@muarachmann yeah thank you

Reply:
@muarachmann did you check?

Reply:
sorry @RAJAGOPALAN-GANGADHARAN have been a lil busy. i see the PR was #1200 and not this wil test that and leave a review